atomic_reactor/plugins/pre_pull_base_image.py

```diff
@@ -34,8 +35,14 @@ def __init__(self, tasker, workflow, parent_registry=None, parent_registry_insec
         # call parent constructor
         super(PullBaseImagePlugin, self).__init__(tasker, workflow)
 
-        self.parent_registry = parent_registry
-        self.parent_registry_insecure = parent_registry_insecure
+        source_registry = get_source_registry(self.workflow, {
+            'uri': parent_registry, 'insecure': parent_registry_insecure})
+
+        if source_registry.get('uri'):
+            self.parent_registry = source_registry['uri'].docker_uri
+        else:
+            self.parent_registry = None
+        self.parent_registry_insecure = source_registry['insecure']
```

Comment:
Should this line be moved to the if-block above and `parent_registry_insecure` be set to None in else-block?

Reply:
well for consistency, insecure should be True or False and not None

Reply:
Sure, pick one.